src/window.h

```diff
@@ -495,6 +495,12 @@ wset_next_buffers (struct window *w, Lisp_Object val)
   w->next_buffers = val;
 }
 
+INLINE void
+wset_dedicated (struct window *w, Lisp_Object val)
+{
```

Comment:
You can actually just get rid of this accessor function, now that everything has a proper binding.

Reply:
Fixed already